app/models/section.rb

```diff
@@ -36,12 +36,17 @@ def self.find(manual, section_uuid, published: false)
   def initialize(manual:, uuid:, previous_edition: nil, latest_edition: nil)
     @slug_generator = SlugGenerator.new(prefix: manual.slug)
     @uuid = uuid
-    @editions = [previous_edition, latest_edition].compact.uniq
-    if @editions.empty?
-      edition = SectionEdition.new(state: "draft", version_number: 1, section_uuid: uuid)
-      @editions.push(edition)
+
+    @previous_edition = previous_edition
+    @latest_edition = latest_edition
+
+    if @previous_edition == @latest_edition
+      @previous_edition = nil
+    end
```

Comment:
I've updated the commit note to explain this code:

> Note that I'm continuing to avoid storing a `previous_edition` and
> `latest_edition` in `Section#initialize` when they're both the same
> object. I haven't tested this behaviour as I'm hoping/expecting it to go
> away in the near future.